DP3TApp/Screens/Vaccination/NSVaccinationInfoViewController.swift

```diff
@@ -90,36 +92,60 @@ private class VaccinationInfoContentView: NSSimpleModuleBaseView {
     // MARK: - Setup
 
     private func setup() {
+        let bookNowLabel = NSLabel(.textBold)
+        bookNowLabel.text = "vaccination_impf_check_title".ub_localized
+
+        contentView.addSpacerView(NSPadding.large + NSPadding.small)
+        contentView.addArrangedView(bookNowLabel, insets: .zero)
+
+        let bookNowTextLabel = NSLabel(.textLight)
+        bookNowTextLabel.text = "vaccination_impf_check_info_text".ub_localized
```

Comment:
this should be
config?.vaccinationBookingInfo.value?.impfcheckText ?? "vaccination_impf_check_info_text".ub_localized